VultisigApp/VultisigApp/Views/Send/SendCryptoVerifyView.swift

```diff
@@ -33,35 +34,30 @@ struct SendCryptoVerifyView: View {
             sendCryptoVerifyViewModel.isLoading = false
         }
         .onAppear {
-            if (tx.coin.chainType == .EVM) {
-                isLoading = true
-                Task{
-                    do {
-                        blowfishResponse = try await sendCryptoVerifyViewModel.blowfishEVMTransactionScan(tx: tx)
-                        isLoading = false
-                    } catch {
-                        print(error.localizedDescription)
-                        isLoading = false
-                    }
+            isLoading = true
```

Comment:
All the methods can be moved to a separate method (we usually set ut as setData()) to make the code cleaner.

Reply:
done